src/app/services/formula.service.ts

```diff
@@ -56,15 +56,15 @@ export class FormulaService {
   }
 
   /**
-   * Returns the result when value is deducted by percent
+   * Finds the net when net * (1+percentage) = gross value
    * @function
-   * @param {number} value Original value
+   * @param {number} value Gross value
    * @param {number} percentage percentage to deduct (10 = 10%)
    * @memberof FormulaService
    * @returns number
    */
   deductPercentage(value: number, percentage: number): number {
```

Comment:
The name `deductPercentage` is a bit misleading given the new formula.  Could we call it something like `removeTax` or `grossToNet` instead?

Reply:
Changed to `inversePercentage` in `597c562`.